rosidl_typesupport_introspection_cpp/cmake/rosidl_typesupport_introspection_cpp_generate_interfaces.cmake

```diff
@@ -62,6 +62,8 @@ add_library(${rosidl_generate_interfaces_TARGET}${_target_suffix} SHARED ${_gene
 if(WIN32)
   target_compile_definitions(${rosidl_generate_interfaces_TARGET}${_target_suffix}
     PRIVATE "ROSIDL_TSI_CPP_BUILDING_DLL")
+  target_compile_definitions(${rosidl_generate_interfaces_TARGET}${_target_suffix}
+    PRIVATE "ROSIDL_BUILDING_DLL")
```

Comment:
According to the CMake docs it looks like that you can specify both definition in a single invocation.